cosmwasm/packages/sgx-vm/src/errors/enclave.rs

```diff
@@ -15,6 +15,8 @@ pub enum EnclaveError {
         status: sgx_types::sgx_status_t,
         backtrace: Backtrace,
     },
+    #[snafu(display("Too many queries, please try again"))]
+    EnclaveBusy {},
 }
```

Comment:
Users will see this string if a query times out on enclave access. Is this ok?

Reply:
Better to do "Too many queries, node is processing block" - more informative for node runners

Reply:
it's not really related to blocks. A query won't be waiting there if a block is running